osmosis-1/osmosis.zone_assets.json

```diff
@@ -4445,7 +4445,7 @@
       "chain_name": "composable",
       "base_denom": "ibc/9D5DA3720001F91DD76B8F609A93F96688EC8185B54BF9A1A1450EB34FF2D912",
       "path": "transfer/channel-1279/transfer/channel-71/ATeTQcUkWGs7AZ15mCiFUWCW9EUL7KpDZEHCN1Y8pump",
-      "osmosis_verified": false,
+      "osmosis_verified": true,
```

Comment:
This property is for the the wrong asset ($WHINE), but I think you mean for it to be $NIBI